modules/ROOT/pages/anypoint-connectors.adoc

```diff
@@ -2,45 +2,51 @@
+
+The list of https://www.mulesoft.com/exchange/?search=Mule%203&type=connector[Mule 3 connectors in Exchange] 
+provide descriptions and helpful links that help you understand how each works.
+
+In addition, Exchange provides Mule 3 https://www.mulesoft.com/exchange/?search=Mule%203&type=template[templates] and https://www.mulesoft.com/exchange/?search=Mule%203&type=example[examples] that let you jump start application development.
+
 
 [IMPORTANT]
 ====
-For connectors that are compliant with Mule 4, see xref:connectors::index.adoc[Connectors and Modules (for Mule 4)].
+For Mule 4 connectors, see xref:connectors::index.adoc[Connectors and Modules (for Mule 4)].
 ====
 
-== Installing
+== Install a Connector
 
 . In Anypoint Studio, click the Exchange icon in the Studio taskbar.
 . Click Login in Anypoint Exchange.
 . Search for the connector and click Install.
 . Follow the prompts to install the connector.
 
-When Studio has an update, a message displays in the lower right corner, which you can click to install the update.
-
-== Using a Connector
+== How to Use a Connector
 
-You can configure and use a connector in Anypoint Studio following its particular setup guidelines, respecting the functionality and limitations of the API or protocol the connector supports. For a tutorial on basic connector usage, see our xref:general:getting-started:anypoint-connector.adoc[Anypoint Connector quickstart].
+For a tutorial on basic connector usage, see the xref:general:getting-started:anypoint-connector.adoc[Anypoint Connector quickstart].
 
-Import additional connectors into your Studio instance, or manually create applications using connectors, that is outside of the Studio development environment, by adding them as Maven dependencies or including the required connector libraries with your project.
-
-The following shows the relationship of connectors to other components in the Anypoint Platform:
-
-image::anypointconnectorgraphic.png[AnypointConnectorGraphic]
-
-*Components*
-
-* xref:3.8@connector-devkit::index.adoc[Anypoint Connector DevKit]
-* xref:web-service-consumer.adoc[Web Service Consumer]
-* xref:http-connector.adoc[HTTP Connector]
-* https://www.anypoint.mulesoft.com/exchange/[Anypoint Exchange]
-* xref:data-gateway::installing-anypoint-data-gateway.adoc[Anypoint Data Gateway]
+You can import additional connectors into your Studio project, or manually create applications using connectors, outside of the Studio development environment, by adding them as Maven dependencies, or including the required connector libraries with your project.
 
 == Connector Support Categories
 
-Connectors fall into one of the following categories. See https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Connector Support Policy] for more information.
+MuleSoft provides different. See https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Connector Support Policies] depending on which connector you use. 
```

Comment:
Because it's worked in as part of the sentence I would lc connector support policies.